base.css

```diff
@@ -15771,4 +15771,14 @@ body.styledGuildsAsChannels-DNHtg_ #app-mount .guilds-1SWlCJ .guildSeparator-3s6
     transform: translate(0, 0) rotate(0);
     opacity: 1;
   }
-}
\ No newline at end of file
+}
+
+.attachPopout-36hjtN{
+    transform: translateY(-50px);
```

Comment:
Pushes the container up just enough so that it fits entirely on the screen.

Pushing it too high leaves a grey box behind for some reason, so, like... don't do that.